syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -765,6 +719,22 @@ def mean(self, shares, **kwargs):
 
         return result
 
+    @staticmethod
+    def share_combine(tensors_shares):
```

Comment:
what does share_combine and apply to share do?

Reply:
Maybe add a short docstring?